qchem/pennylane_qchem/_version.py

```diff
@@ -16,4 +16,4 @@
 Version number (major.minor.patch[-label])
 """
 
-__version__ = "0.13.0-dev"
+__version__ = "0.14.0-dev"
```

Comment:
Was this missed during the pre-release bump?

Reply:
Yep :) I spotted this and fixed this before making the release though, which was lucky.